cylc/flow/scripts/tui.py

```diff
@@ -15,34 +15,37 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-"""cylc tui WORKFLOW
+"""cylc tui [WORKFLOW]
```

Comment:
If you give a workflow, tui will startup, pre-filtered to just that workflow. Press "E" to unset this filtering an view all workflows.